### PR TITLE
README: Make command examples valid

### PR DESCRIPTION
**Pull request by felder-cl**
Make it easier to run the example shell commands by making the syntax
valid.

### CODE REVIEW

**Comment by felder-cl** [comment]
Would be nice to put the examples in the README through CI too

**Comment by j16r** [comment]
I believe the `$ ` syntax is relatively common in readmes, it indicates a shell entry.

**Comment by felder-cl** [comment]
I noticed it in places as well.  We have mixed instances of using it or not in our codebase (admittedly added by myself).

https://github.com/smartcontractkit/chainlink/tree/develop/tools/docker#adding-new-environment-variables

They continually get in the way, and I find don't add much value.  Suggest still we remove them.  Maybe we get more consensus?

**Comment by felder-cl** [comment]
"Using $ is common to indicate bash commands." Already mentioned above ;)

**Comment by RyanRHall** [comment]
Personally I'd prefer to omit to the `$` -  it makes it easier to copy/paste commands.

File: tools/docker/README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ $$ cldev core # import our testing key and api credentials, then start the node
 In the first terminal:
 
 ```sh
-$ ./compose cldo
+./compose cldo
 #
 # $$ denotes that we're now in the chainlink container
 $$ cldev # cldev without the "core" postfix simply calls the core node cli
```
Comment (RyanRHall) [inline tools/docker/README.md]: remove `$$` and adjust comment

File: tools/docker/README.md [hunk 2]
```diff
@@ -42,10 +42,10 @@ The `cld` command will bring up the services that a chainlink node needs to conn
 This enables a user to make quick changes on either the container or the host, run `cldev` within the attached container, check the new behaviour of the re-built node, and repeat this process until the desired results are achieved.
 
 ```sh
-$ ./compose cld
+./compose cld
 #
 # $$ denotes that we're now in the chainlink container
```
Comment (RyanRHall) [inline tools/docker/README.md]: adjust comment

**Comment by felder-cl** [comment]
This PR should not slow anyone down.  Its a quick win.

**Pull request merged**